h5py_wrapper/wrapper.py

```diff
@@ -259,6 +259,8 @@ def _load_dataset(f, lazy=False):
     else:
         try:
             value_type = f.attrs['_value_type']
+            if isinstance(value_type, bytes):
```

Comment:
does this need to be in the try block? it should only contain the minimal part of the function that we want to catch key errors from.